js/treeSortable.js

```diff
@@ -322,24 +322,38 @@ function TreeSortable() {
                              */
                             if ($sibling.length) {
                                 const distance = getDistance($(this).get(0), $sibling.get(0));
+                                var ThisParent=$(this).getParent()?$(this).getParent():$(this).getRootChild()
```

Comment:
Use `const` instead of  `var`. For the variable naming I've used camelCase in all the places, so please update the variable name.

```suggestion
                                const thisParent = $(this).getParent() || $(this).getRootChild()
```